### ISSUE TITLE
WPF NET Core Demo

### ISSUE THREAD

**Issue by abrasat** [dapplo/Dapplo.Microsoft.Extensions.Hosting#20]
Thank you for this useful repository.
Could you please provide some samples about using the extensions with some more complex WPF .NET Core applications using MVVM? For instance how can view models be registered and assigned to the different views from some WPF application?

**Comment by Lakritzator** [comment]
It was never the intention of this repository to provide a MVVM pattern directly, this was supposed to be covered with different libraries like Caliburn.Micro, ReactiveUI and others. But there is basic MVVM functionalty you can use for small applications, brought to you by @jorisvergeer, but for more I do advice you to look at others.

This repository was supposed to bring you a way to use a UI on top of the Hosting extensions which are provided by Microsoft. This gives you a possibility to use services and UI together.

You can have a look at the sample code, in the samples directory, for[ Caliburn.Micro](https://github.com/dapplo/Dapplo.Microsoft.Extensions.Hosting/tree/master/samples/Dapplo.Hosting.Sample.CaliburnMicroDemo) or [ReactiveUI](https://github.com/dapplo/Dapplo.Microsoft.Extensions.Hosting/tree/master/samples/Dapplo.Hosting.Sample.ReactiveDemo), this should help you get started. 

Hope this answers your question?

**Comment by abrasat** [comment]
Yes,thx. The explanation and the links will help me getting started
